Standards/scs-0100-v2-flavor-naming.md

```diff
@@ -384,9 +384,9 @@ capabilities. Flavors may over-deliver ...)
 #### Examples
 
 - SCS-2C-4-10 <- may or may not support HW virtualization in VMs
-- SCS-2C-4-10_kvm_**hwv** <- kvm with enabled nested virtualization
+- SCS-2C-4-10*kvm***hwv** <- kvm with enabled nested virtualization
 - SCS-2C-4-10\_**hwv** <- not recommended, but allowed
-- SCS-2C-4-10\_bms\_**hwv** <- better: bare metal with HW virt support (VMX on intel, SVM on AMD, ...)
+- SCS-2C-4-10_bms\_**hwv** <- better: bare metal with HW virt support (VMX on intel, SVM on AMD, ...)
 - ~~SCS-2C-4-10\_**hwv**\_xen~~ <- illegal, wrong ordering
```

Comment:
These changes are wrong. The symbols here are not for formatting, but part of the flavor names. If this is a problem, then the names have to be put as code using back-ticks.

Reply:
I think I made a mistake here -- the underscores are part of the name whereas the double asterisks are meant for emphasis. So the code block doesn't work. Maybe we need to escape all underscores?